### PR TITLE
Added support for Suse

### PR DESCRIPTION
**Pull request by jgrumboe**

### CODE REVIEW

File: manifests/install/package.pp [hunk 1]
```diff
@@ -81,6 +81,18 @@
         }
       }
 
+      'Suse'   : {
+        if $vmwaretools::redhat_install_devel == true {
```
Comment (craigwatson) [inline manifests/install/package.pp]: I'm unsure about this - perhaps the parameter should be renamed to something more generic? Maybe `install_devel`?

**Comment by jgrumboe** [comment]
Hmm.. good point. I just did a copy/paste.
I will look into this.

**Comment by jgrumboe** [comment]
I renamed <code>redhat_install_devel</code> to <code>install_devel</code>. First i missed one occurence of the parameter but now Travis-Build is also OK.

**Comment by craigwatson** [comment]
Thanks, much appreciated :)

**Pull request merged**